mantidproject/mantid
# Modify SliceViewer to add support for sorting by I/sigma

These changes allow the user to sort by the  I/sigma column in the SliceViewer and any other column that was previously disabled.

**To test:**

Some data for testing can be found under Scratch/Sam. The MD workspace is md.nxs, the peaks workspace is satellite_refined.nxs.

 - Load an MD workspace
 - Load a peaks workspace
 - Right click -> Open the slice viewer on the MD workspace
 - Overlay peaks workspace on the slice viewer
 - Sort by the I/sigma column.

Also check that you can add/erase peaks and click rows to zoom to a peak and that this a) does not crash mantid and b) does the correct thing (i.e. the sort order does not break these features).

Fixes #20441

**Release Notes** 
[See here](https://github.com/mantidproject/mantid/pull/21511/commits/c1de0eb0ba80c01cefece822545b86eee6d21e1a)

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
